### ISSUE TITLE
Problem with automagically saving subplot

### ISSUE THREAD

**Issue by RoosBressers** [allegroai/clearml#662]
This problem was discussed on slack.

In my pytorch code I made a graph to show the loss and accuracy using the following lines:
`fig, axs = plt.subplots(2, 1)
axs[1].plot(range(epoch), accuracy_train, range(epoch), accuracy_valid)
axs[1].legend(['Training accuracy', 'Validation accuracy'])
axs[0].plot(range(epoch), loss_train, range(epoch), loss_valid)
axs[0].legend(['Training loss', 'Validation loss'])
plt.show()`

The result on my local device can be seen in the first image below (the nice plot with two subplots). This plot is automagically saved in clearml, the result that I see in the web UI is in the second attached image below. This is clearly not the same graph.

There are several (unexpected?) things happening in the output in the web UI:
- The plot is not a subplot anymore but there are 'child' labels added on the right. 
- It only contains the information from the bottom subplot of the original. 
- The y-axis has two different sets of labels and the x-axis labels are very far away from the axis. 
It seems to be trying to combine the two subplots in one?

My clearml version is 1.3.2

![image](https://user-images.githubusercontent.com/47984997/165268428-e62153ea-8abf-4bf2-8811-db7523c221d1.png)
![image](https://user-images.githubusercontent.com/47984997/165268454-0ed4e434-23ad-4dde-8a1a-dd9311712524.png)

**Comment by DavidNativ** [comment]
Hello @RoosBressers 
The bug acquiring subplots has been fixed in the 1.4.0 version, released a week ago. You should now get the same nice two subplots in the WebApp :)
